### PR TITLE
SORQA-1035 Implement "Check Vaccination and Vaccination Status of cas…

### PR DESCRIPTION
**Pull request by mk-sgent**

### CODE REVIEW

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12287

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12287

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12287

**Comment by mk-sgent** [comment]
Jenkins jobs:
https://jenkins.sormas.netzlink.com/view/SORMAS-E2eTests/job/sormas-RunE2eTests-branch/3082/allure/#suites/03ef468fbeb9479f23a936de20f404be/ab05750b09be0a82/

image

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12287

File: sormas-e2e-tests/src/main/java/org/sormas/e2etests/enums/immunizations/VaccineNameValues.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+import java.util.Random;
+import lombok.Getter;
+
+@Getter
+public enum VaccineNameValues {
+  COMIRNATY("Comirnaty (COVID-19-mRNA Impfstoff)"),
+  MRNA_BIVALENT_BA_1_BIONTECH_PFIZER("mRNA/bivalent BA.1 (BioNTech/Pfizer)"),
+  MRNA_BIVALENT_BA_4_5_BIONTECH_PFIZER("mRNA/bivalent BA.4/5 (BioNTech/Pfizer)"),
+  COVID_19_Impfstoff_Moderna("COVID-19 Impfstoff Moderna (mRNA-Impfstoff)"),
```
Comment (hms-sgent) [inline sormas-e2e-tests/src/main/java/org/sormas/e2etests/enums/immunizations/VaccineNameValues.java:29]: Please change this ENUM: "COVID_19_Impfstoff_Moderna" to uppercase

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12287

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=12287

**Pull request merged**